kurl_util/Makefile

```diff
@@ -85,6 +85,15 @@ bin/vendorflights:
 bin/pvmigrate: cmd/pvmigrate/main.go
 	go build ${LDFLAGS} -o bin/pvmigrate cmd/pvmigrate/main.go
 
+bin/rook-pv-migrator:
+	rm -rf .tmp/persistent-volume-migrator
+	mkdir -p .tmp/persistent-volume-migrator
+	cd .tmp/persistent-volume-migrator \
+		&& git clone -b csi-source-sc https://github.com/replicatedhq/persistent-volume-migrator.git . \
+		&& go build -o ../../bin/rook-pv-migrator
```

Comment:
this is still so weird to me

I can't wait until we can use the upstream binary